src/main/java/com/kodilla/ecommercee/CartNotFoundException.java

```diff
@@ -0,0 +1,4 @@
+package com.kodilla.ecommercee;
+
+public class CartNotFoundException extends Exception {
+}
```

Comment:
staraj sie uzywac unchecked exception niz checked exception'ow

Reply:
dodatkowo jesli ten exception poleci, to serwer zwroci 500. Mozna wskazac springowi zeby zwocil wlasciwy error code np
```
...
import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.ResponseStatus;

@ResponseStatus(code = HttpStatus.NOT_FOUND)
public class MyException extends RuntimeException{
```

Reply:
wtedy serwer odpowie 404
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

Reply:
Poprawiłem według twoich wskazówek.